cypress.config.js

```diff
@@ -0,0 +1,26 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig ({
+  projectId: "kvars8",
+  viewportWidth: 1920,
+  viewportHeight: 1080,
+  env: {
+    API_URL: "http://localhost:3001",
```

Comment:
Para algumas strings você está usando aspas simples e para outras aspas duplas.

Definir um padrão e seguí-lo é importante para que o código seja consistente.

Portanto, recomendo definir se vai usar um ou outro e usar o que for definido.

A propósito, é possível configurar uma regra no ESLint para isso também.